src/data/code-sample-definitions/thermostats.yaml

```
@@ -29,13 +29,28 @@
             connected_account_id: 123e4567-e89b-12d3-a456-426614174000
             workspace_id: 123e4567-e89b-12d3-a456-426614174000
             created_at: '2024-04-05T07:57:05.323Z'
+- title: Set HVAC mode
+  description: Specify the desired heating and cooling set points in °C or °F.
+  request:
+    path: /thermostats/set_hvac_mode
+    parameters:
+      device_id: 123e4567-e89b-12d3-a456-426614174000
+      hvac_mode_setting: 'heat_cool'
+      heating_set_point_celsius: 20
+      cooling_set_point_celsius: 25
+  response:
+    body:
+      action_attempt:
+        action_attempt_id: 123e4567-e89b-12d3-a456-426614174000
+        status: pending
```

Found one more thing, the SDKs should wait be default so we should document this as. I think this is generally more helpful anyway, since pending action attempts are "boring" while resolved one will contain an interesting result in some cases.

```suggestion
        status: success
```

---

@razor-x OK, but I think I need to understand better, then, why we have instructions in the docs that specifically say to poll the action attempt until it returns as success. This feels like a broader discussion.